src/Illuminate/Database/Eloquent/Factories/HasRealTimeFactory.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Illuminate\Database\Eloquent\Factories;
+
+trait HasRealTimeFactory
+{
+    use HasFactory;
+
+    /**
+     * Create a new factory instance for the model.
+     *
+     * @return \Illuminate\Database\Eloquent\Factories\Factory<static>
+     */
+    protected static function newFactory()
+    {
+        return (new RealTimeFactory)
```

Comment:
Could you build RealTimeFactory from the container? I can already see wanting to add options to the guessable types. Another option is to create a static method to allow us to set those in user-land, but I think that's much less robust.

Reply:
At this point, consider using an actual factory, or passing the individual column values in the `create` method.

Reply:
It is easy enough to just create a new application-level trait for HasRealTimeFactory and also extend the RealTimeFactory class to add to the list.

I would at least suggest providing the ability to add to this list in userland, lest GitHub be inundated with weekly PRs to add "family_name", "telephone_number", etc.

Reply:
We'll leave this for Taylor to look at.